File: src/main/java/com/example/demo/impl/UserMapper.java
package com.example.demo.impl;

import com.example.demo.entity.UserBean;
import org.apache.ibatis.annotations.Mapper;

@Mapper
public interface UserMapper {

    UserBean login(String phone, String password);

    void register(String phone, String password);

    UserBean findUser(String phone);

}

File: src/main/java/com/example/demo/impl/UserMapper.kt
package com.example.demo.impl

import com.example.demo.entity.UserBean
import org.apache.ibatis.annotations.Mapper


/**
 * @Author Mr.Wu
 * @Date create on 2020/1/11
 * @DES
 */
@Mapper
interface UserMapper {
    fun login(phone: String?, password: String?): UserBean?
    fun register(phone: String?, password: String?)
    fun findUser(phone: String?): UserBean?
}